packages/itwin/map-layers/src/ui/widget/FeatureInfoWidget.tsx

```diff
@@ -60,14 +62,26 @@ export function MapFeatureInfoWidget({ featureInfoOpts }: MapFeatureInfoWidgetPr
 
   }, [featureInfoOpts?.showLoadProgressAnimation]);
 
+  const handleResize = React.useCallback((w: number, h: number) => {
+    setSize({ width: w, height: h });
+  }, []);
+
   if (loadingData) {
     return (<FillCentered><ProgressRadial indeterminate={true}></ProgressRadial></FillCentered>);
   } else if (!hasData) {
     return (<FillCentered><span><i>{noRecordsMessage}</i></span></FillCentered>);
   } else {
     if (dataProvider.current)
-      return (<PropertyGrid dataProvider={dataProvider.current} orientation={Orientation.Vertical} // eslint-disable-line deprecation/deprecation
-        isPropertySelectionEnabled={featureInfoOpts?.propertyGridOptions?.isPropertySelectionEnabled} />);
+      return (
+        <ResizableContainerObserver onResize={handleResize}>
+          <VirtualizedPropertyGridWithDataProvider
```

Comment:
are you going to pull in the data provider fix grigas mentioned yesterday?

Reply:
No. I have my own provider, and made my own fix.